contracts/sidechain/peripheral/SidechainClaimZap.sol

```diff
@@ -0,0 +1,243 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.11;
+
+import { IERC20 } from "@openzeppelin/contracts-0.8/token/ERC20/IERC20.sol";
+import { SafeERC20 } from "@openzeppelin/contracts-0.8/token/ERC20/utils/SafeERC20.sol";
+import { AuraMath } from "../../utils/AuraMath.sol";
+import { IRewardStaking } from "../../interfaces/IRewardStaking.sol";
+import { IRewardPool4626 } from "../../interfaces/IRewardPool4626.sol";
+import { IAuraOFT } from "../interfaces/IAuraOFT.sol";
+
+/**
+ * @title   SidechainClaimZap
+ * @author  AuraFinance
+ * @notice  Claim zap to bundle various reward claims
+ * @dev     Claims from all pools, Bridges/Locks to L1 if Wanted and compounds if needed.
+ *
+ */
+contract SidechainClaimZap {
```

Comment:
@hephyrius
- SidechainClaimZap has 2 lint issues 
unused variable data

L153 
(bool sent, bytes memory data) = payable(msg.sender).call{ value: address(this).balance }("");

Consider refactor to 
(bool result, ) = payable(msg.sender).call{ value: address(this).balance }("");

fn could be change to pure

L162 
_callOptions